src/Common/Admin/Conditional_Content/Black_Friday.php

```diff
@@ -0,0 +1,177 @@
+<?php
+namespace TEC\Common\Admin\Conditional_Content;
+
+use TEC\Common\Admin\Settings_Sidebar;
+use Tribe\Utils\Element_Attributes as Attributes;
+use TEC\Common\Admin\Entities\Link;
+use TEC\Common\Admin\Entities\Image;
+use TEC\Common\Admin\Settings_Section;
+use Tribe\Utils\Date_I18n;
+
+/**
+ * Set up for Black Friday promo.
+ *
+ * @since TBD
+ */
+class Black_Friday extends Datetime_Conditional_Abstract {
+	/**
+	 * @inheritdoc
+	 */
+	protected string $slug = 'black_friday';
+
+	/**
+	 * @inheritdoc
+	 */
+	protected string $start_date = 'fourth Thursday of November';
+
+	/**
+	 * @inheritdoc
+	 */
+	protected string $end_date = 'November 30th';
+
+	/**
+	 * @inheritdoc
+	 */
+	public function hook(): void {
+		add_action( 'tec_settings_sidebar_start', [ $this, 'include_sidebar_section' ] );
+		add_action( 'tribe_settings_below_tabs', [ $this, 'include_tickets_settings_section' ] );
+	}
+
+	/**
+	 * @inheritdoc
+	 */
+	protected function get_start_time(): ?Date_I18n {
+		$date = parent::get_start_time();
+		if ( null === $date ) {
+			return null;
+		}
+
+		$date = $date->modify( '-3 days' );
+
+		return $date;
+	}
+
+	protected function should_display(): bool {
+		return true;
+	}
```

Comment:
I am fully aware this is here. For In branch QA purposes, will be removed once we pass QA.

Reply:
Would be nice if we were able to QA appearance logic as well. A bit more complicated for sure, but it's possible to fake the current time even for QA